common/bond.js

```diff
@@ -23,35 +23,41 @@ const bondBaseUrl = () => config.bondBaseUrl;
 // At some point we expect to have a more sophisticated way to do this, but for now, we have to do it this way.
 function determineBondProvider(urlString) {
     const url = URL.parse(urlString);
+
     if (DATASTAGE_NAMESPACES.includes(url.hostname.toLowerCase())) {
         return BondProviders.FENCE;
-    } else if (url.host.endsWith('.humancellatlas.org')) {
-        return BondProviders.HCA;
-    } else if (hasJadeDataRepoHost(url)) {
-        return BondProviders.JADE_DATA_REPO;
-    } else {
-        return BondProviders.default;
     }
+
+    if (ANVIL_NAMESPACE === url.hostname.toLowerCase()) {
+        return BondProviders.ANVIL;
+    }
+
+    if (url.host.endsWith('.humancellatlas.org')) {
+        return;
+    }
+
+    if (hasJadeDataRepoHost(url)) {
+        return;
+    }
```

Comment:
why not combine this into one `if` statement?
```
if (url.host.endsWith('.humancellatlas.org') || hasJadeDataRepoHost(url)) {
        return;
}

Reply:
I realize [guard clauses](https://www.google.com/search?q=javascript+guard+clause) are usually aimed at nested conditionals, but I also consider lots of `|| && ()` mixed together in a conditional soup harder to read than multiple lines.

Reply:
TOL If you think it's likely that more `||`s or `&&`s will be added, maybe a helper function instead?